File: simpleclaw_mobile/lib/config/app_config.dart
class AppConfig {
  AppConfig._();

  static const String apiBaseUrl = String.fromEnvironment(
    'API_BASE_URL',
    defaultValue: 'https://install-openclow.ru/api',
  );

  static const String googleClientId =
      '1568931022-iil9topt7v2n8p6m97crp4tc410800mf.apps.googleusercontent.com';

  static const int deployPollIntervalMs = 5000;

  static const String supportEmail = 'tarasov.slavas2@gmail.com';

  static const String frontendUrl = 'https://install-openclow.ru';

  static const String demoVideoUrl = 'https://install-openclow.ru/demo.mp4';
}

File: simpleclaw-rn/src/config/appConfig.ts
export const AppConfig = {
  apiBaseUrl: process.env.EXPO_PUBLIC_API_BASE_URL || 'https://install-openclow.ru/api',
  googleClientId: '1568931022-iil9topt7v2n8p6m97crp4tc410800mf.apps.googleusercontent.com',
  deployPollIntervalMs: 5000,
  supportEmail: 'tarasov.slavas2@gmail.com',
  frontendUrl: 'https://install-openclow.ru',
  demoVideoUrl: 'https://install-openclow.ru/demo.mp4',
  yookassaShopId: process.env.EXPO_PUBLIC_YOOKASSA_SHOP_ID || '',
  yookassaClientKey: process.env.EXPO_PUBLIC_YOOKASSA_CLIENT_KEY || '',
  subscriptionPriceRub: '2999',
} as const;
